bundle/compliance/policy/gcp/iam/is_user_with_multiple_roles.rego

```diff
@@ -0,0 +1,15 @@
+package compliance.policy.gcp.iam.is_user_with_multiple_roles
+
+import data.compliance.policy.gcp.data_adapter
+import future.keywords.if
+
+is_user_with_multiple_roles(role_a, role_b) if {
```

Comment:
this (and the original version) is failing when adding `count(m) == 0`) which is required. moved to draft to fix it